README.md

```diff
@@ -9,68 +9,301 @@
 [![VSCode Plugin][vscode-shield]][vscode-url]
 [![Twitter Follow][twitter-shield]][twitter-url]
 
-# Sourcery for VS Code
+![Sourcery](https://raw.githubusercontent.com/sourcery-ai/sourcery-vscode/master/assets/Sourcery_Logo_VS_Code_Description.svg)
 
-Sourcery is your personal coding assistant that helps you write better, cleaner, Python code. It suggests refactorings on the fly to instantly improve your code. 
+Sourcery is the **pair programmer** who will help you improve your code anytime you're working in Python. It **reviews and refactors** your code automatically so you can spend more time focused on writing new code and less time cleaning things up.
 
-![Sourcery in VS Code](https://raw.githubusercontent.com/sourcery-ai/sourcery-vscode/master/assets/VS_Code_Refactorings.gif)
+![Sourcery in VS Code](https://raw.githubusercontent.com/sourcery-ai/sourcery-vscode/master/assets/Sourcery_VS_Code_Example.gif)
 
 ---
 
-### Using Sourcery
+## Sourcery
 
-Sourcery runs in the background whenever you're working on Python files. Keep an eye out for sections of your code underlined in blue - that means Sourcery has a suggestion!
+Sourcery is a VS Code extension to help make all of your Python code cleaner and more readable. Quickly find areas **where your code could be refactored**, see **instant suggestions for improvement**, and understand how new changes impact your **code quality**. 
 
-Hover over a Sourcery suggestion to see a description of what Sourcery wants you to change along with the diff of the proposed changes. Then either use the quickfix menu (`Ctrl+.` or `Cmd+.` for most people) or click on the lightbulb icon to bring up the Sourcery menu and choose to accept or reject the changes.
+Here are some of the features Sourcery offers to help improve your code:
 
-Sourcery also provides quality metrics for every function you write (plus warnings for really bad code).
+- [Real-time refactoring suggestions](#real-time-refactoring-suggestions)
+- [Continuous code quality feedback](#continuous-code-quality-feedback)
+- [Multi-file analysis](#multi-file-analysis)
+- [Duplicate code detection](#duplicate-code-detection)
+- [GitHub Pull Request reviews](#github-pull-request-review)
+- [Sourcery CLI, CI, & Pre-Commit Hook options](#sourcery-cli)
 
-![Quality Metrics](https://raw.githubusercontent.com/sourcery-ai/sourcery-vscode/master/assets/Hover_Over_Metrics_VS_Code.gif)
+To start using Sourcery on your code, check out our [Getting Started guide](https://docs.sourcery.ai/).
```

Comment:
This doesn't link to the getting started page

Reply:
Good point - fixed